apps/line_clock/settings.js

```diff
@@ -0,0 +1,39 @@
+(function(back) {
+    const SETTINGS_FILE = "line_clock.setting.json";
+
+    // initialize with default settings...
+    const storage = require('Storage')
+    let settings = {
+      showLock: true,
+      showMinute: true,
+    };
+    let saved_settings = storage.readJSON(SETTINGS_FILE, 1) || settings;
+    for (const key in saved_settings) {
+      settings[key] = saved_settings[key]
+    }
+
+    function save() {
+      storage.write(SETTINGS_FILE, settings)
+    }
+
+    E.showMenu({
+      '': { 'title': 'Line Clock' },
+      '< Back': back,
+      'Show Lock': {
+        value: settings.showLock,
+        format: () => (settings.showLock ? 'Yes' : 'No'),
```

Comment:
There's a default boolean format with empty/filled checkbox. 

I'll take thsse Yes/No formattings out upon merging :+1:

See recent PR https://github.com/espruino/BangleApps/pull/3149 by @bobrippling for reference.